src/Raven.Studio/wwwroot/Content/css/pages/database-settings.less

```diff
@@ -0,0 +1,185 @@
+#database-settings-view {
+    
+    @source-item-color: @database-settings-source-item;
+    @source-item-color-basic: @database-settings-source-item-basic;
+    @highlight-key-color: @database-origin-key-color;
+    
+    .all-entries {
+        overflow-x: hidden;
+        top: 50px;
```

Comment:
why?

Reply:
Vertical scroll needs _position: absolute_
If I use _position: relative_ then vertical scroll is not in place.
So adding _top: 50_ solved it.
(it is for the start location of the entries, below the titles, in edit mode)